packages/suite/src/support/messages.ts

```diff
@@ -7374,6 +7374,28 @@ export default defineMessages({
         id: 'TR_STOP',
         defaultMessage: 'Stop',
     },
+    TR_AUTO_STOP: {
+        id: 'TR_AUTO_STOP',
+        defaultMessage: 'Stop later',
```

Comment:
nitpick: I believe that just "Stop" can work fine here, would be less confusing and looks nicer in small round - what do you think @hynek-jina ?

Reply:
Make sense..

Reply:
done [3d31dcc](https://github.com/trezor/trezor-suite/pull/9095/commits/3d31dcca5dc53cc00b774e57edf766188b24b791)